### PR TITLE
GitHub Action should add expire header to pipeline execution requests

### PR DESCRIPTION
**Pull request by gdelgadot**
Signed-off-by: Gloria Delgado <gloriad@vmware.com>

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -415,10 +408,15 @@ export async function createPipeline(config: Config): Promise<string> {
     core.debug(`Sending pipeline: ${util.inspect(pipeline)}`)
     //TODO: Define and replace different placeholders: e.g. for values, content folders (goss, jmeter), etc.
 
+    const expiresAfter = moment()
+      .add(config.pipelineDuration * 1000, "s")
+      .format("[Date:] ddd, D MMM YYYY HH:mm:ss z")
```
Comment (mpermar) [inline src/main.ts:413]: I don't think this format is right. The `Date` part is not part of a date. A date in RFC9110 looks more like `Tue, 22 Nov 2022 12:25:13 GMT`.

Comment (gdelgadot) [reply src/main.ts]: done

File: dist/index.js [hunk 2]
```diff
@@ -616,10 +613,14 @@ function createPipeline(config) {
             yield validatePipeline(pipeline);
             core.debug(`Sending pipeline: ${util_1.default.inspect(pipeline)}`);
             //TODO: Define and replace different placeholders: e.g. for values, content folders (goss, jmeter), etc.
+            const expiresAfter = (0, moment_1.default)()
+                .add(config.pipelineDuration * 1000, "s")
+                .format("ddd, D MMM YYYY HH:mm:ss z");
```
Comment (mpermar) [inline dist/index.js:618]:
```suggestion
                .format("ddd, DD MMM YYYY HH:mm:ss z");
```

`DD` for days to be 01, 02, ..

Comment (gdelgadot) [reply dist/index.js]: done

**Review by mpermar — APPROVED** [review]

**Pull request merged**